<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QueryStudent</class>
 <widget class="QDialog" name="QueryStudent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>916</width>
    <height>533</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="2">
      <widget class="QComboBox" name="comboBox">
       <item>
        <property name="text">
         <string>学生编号</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>学生姓名</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QPushButton" name="pushButton_select">
       <property name="text">
        <string>查询</string>
       </property>
      </widget>
     </item>
     <item row="0" column="6">
      <widget class="QPushButton" name="pushButton_close">
       <property name="text">
        <string>退出</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>查询关键字:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLineEdit" name="lineEdit_key"/>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>查询方式: </string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="tableView"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_close</sender>
   <signal>clicked()</signal>
   <receiver>QueryStudent</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>857</x>
     <y>26</y>
    </hint>
    <hint type="destinationlabel">
     <x>457</x>
     <y>266</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
